packages/forma-36-react-components/src/components/Card/AssetCard/AssetCard.tsx

```diff
@@ -32,6 +33,10 @@ export interface AssetCardProps extends BaseCardProps {
    * The publish status of the asset
    */
   status?: AssetState;
+  /**
+   * An icon for the status of the entry
+   */
+  statusIcon?: React.ReactNode;
```

Comment:
Since you mentioned you need to support both 'string' and a custom React component, I think this prop should be something like this, right?

`statusIcon?: React.ReactNode | IconProps['icon']`

P.S.: IMO, using `IconProps['icon']` is better than `IconTypes` because if we change the `icon` prop in the future to use something else than IconTypes, we won't need to update this component

Reply:
AssetCard doesn't have an `icon` prop. So I can only add `IconType` there as type definition for `statusIcon`